src/Symfony/Component/Serializer/Normalizer/ArrayDenormalizer.php

```diff
@@ -57,10 +57,12 @@ function ($data) use ($serializer, $class, $format, $context) {
     /**
      * {@inheritdoc}
      */
-    public function supportsDenormalization($data, $type, $format = null)
+    public function supportsDenormalization($data, $type, $format = null/*, array $context = array()*/)
     {
+        $context = 4 === func_num_args() ? func_get_arg(3) : array();
```

Comment:
should be changed to ``func_num_args() > 3 ? func_get_arg(3) : array()`` for consistency (this is always written like that) and for clarity